lib/tchannel.js

```diff
@@ -174,18 +175,19 @@ RingPopTChannel.prototype.protocolPing = function (arg1, arg2, hostInfo, cb) {
     });
 };
 
-RingPopTChannel.prototype.protocolPingReq = function (arg1, arg2, hostInfo, cb) {
+RingPopTChannel.prototype.protocolPingReq = function protocolPingReq(arg1, arg2, hostInfo, cb) {
     var body = safeParse(arg2);
     if (body === null || !body.source || !body.target || !body.changes || !body.checksum) {
         return cb(new Error('need req body with source, target, changes, and checksum'));
     }
 
-    this.ringPop.protocolPingReq({
+    recvPingReq({
```

Comment:
+1

Treating ringpop as a well defined server with endpoints is going to be nicer.